### PR TITLE
Update CMake common, various cleanups

### PR DESCRIPTION
**Pull request by rdumusc**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,50 +1,33 @@
...
 set(VERSION_MINOR 5)
 set(VERSION_PATCH 0)
 set(VERSION_ABI 0)
 
-# Scripts
 set(PROJECT_INCLUDE_NAME dc)
-set(DISPLAYCLUSTER_DESCRIPTION "The DisplayCluster project")
-set(DISPLAYCLUSTER_ISSUES_URL "https://bbpteam.epfl.ch/project/issues/browse/DISCL")
-set(DISPLAYCLUSTER_PACKAGE_URL "https://github.com/BlueBrain/DisplayCluster/releases")
+set(DISPLAYCLUSTER_DESCRIPTION "The DisplayCluster project for driving large display walls")
```
Comment (tribal-tec) [inline CMakeLists.txt]: A collaborative environment/software for driving large display walls?

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,50 +1,33 @@
...
-# Versioning
 set(VERSION_MAJOR 0)
 set(VERSION_MINOR 5)
 set(VERSION_PATCH 0)
 set(VERSION_ABI 0)
 
-# Scripts
 set(PROJECT_INCLUDE_NAME dc)
-set(DISPLAYCLUSTER_DESCRIPTION "The DisplayCluster project")
-set(DISPLAYCLUSTER_ISSUES_URL "https://bbpteam.epfl.ch/project/issues/browse/DISCL")
```
Comment (tribal-tec) [inline CMakeLists.txt]: they are not needed for any of the doxygen targets?

File: src/core/ws/TextInputHandler.h [hunk 3]
```diff
@@ -42,7 +42,7 @@
 
 #include <QObject>
 
-#include <dcWebservice/Handler.h>
+#include "dcWebservice/Handler.h"
```
Comment (tribal-tec) [inline src/core/ws/TextInputHandler.h]: why changed?

File: CMakeLists.txt (hunk 2 above)
Comment (rdumusc) [inline CMakeLists.txt]: Not anymore. The default for the Jekyll-generated index is to point to the github issues page, which also makes it available for external users/contributors.

File: src/core/ws/TextInputHandler.h (hunk 3 above)
Comment (rdumusc) [inline src/core/ws/TextInputHandler.h]: Those are really local includes. They get installed under a different path, namely <dc/webservice/...>. At some point we should improve the situation with namespaces in this project but right now it is the correct syntax.

File: CMakeLists.txt (hunk 1 above)
Comment (rdumusc) [inline CMakeLists.txt]: Yes, it is better. I will update for: 'A collaborative software for driving large display walls'

Comment (tribal-tec) [inline CMakeLists.txt]: Sync with README.md and github repository description.

Comment (rdumusc) [inline CMakeLists.txt]: done

**Pull request merged**